components/Scrollytelling/Scrollytelling.tsx

```diff
@@ -56,20 +56,39 @@ export const Scrollytelling = ({
           className={styles.imageWrapper}
           style={{ scale: animateScale, willChange }}
         >
-          <img
-            src={getProcessedImage(bgImageSrc, '2000x1200', bgImageFocus)}
-            alt={bgImageAlt || ''}
-            width={2000}
-            height={1200}
-            className={styles.image}
-          />
-          <img
-            src={getProcessedImage(bgImageSrc, '900x1600', bgImageFocus)}
-            alt={bgImageAlt || ''}
-            width={900}
-            height={1600}
-            className={styles.imageMobile}
-          />
+          <picture>
```

Comment:
I am curious to why you chose `<picture>` and `<source>` over `<img/>`s `srcset`. Can you tell me more about this decision? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#srcset

Reply:
I've tried srcset in the past, but found the picture with source easier to read. Also, it includes support for device orientation so more flexible.